modin/api/partition_api.py

```diff
@@ -128,13 +129,15 @@ def create_df_from_partitions(partitions, axis):
                     for row in partitions
                 ]
             )
+    # When collecting partitions to NumPy array they will be kept row-wise
     elif axis == 0:
         if isinstance(partitions[0], tuple):
             parts = np.array(
                 [[partition_class(partition, ip=ip)] for ip, partition in partitions]
             )
         else:
             parts = np.array([[partition_class(partition)] for partition in partitions])
+    # When collecting partitions to NumPy array they will be kept column-wise
```

Comment:
This comment tries to explain the "what is done" but not why... a little more info on "why" it is done would be good.
Plus the same comment was applied to `axis is None` case but the code is different.

_I'm being very nosy and nitpicking here becase this API is intented to be an entry point to Modin internals (so it would be something publically used), and anything publically used should be documented better than some internal stuff IMHO_.

Reply:
Fixed.